cmd/build/helmify/static/values.yaml

```diff
@@ -229,6 +229,12 @@ controllerManager:
       # - from:
       #   - ipBlock:
       #       cidr: 0.0.0.0/0
+  disableWebhookOperation: false
+  disableGenerateOperation: true
+  disableAuditOperation: true
```

Comment:
all the variables except `disableWebhookOperation` and `disableGenerateOperation` should be removed since they are not being used.